docs/docs/1.4.1/deployments/how-to-multi-zone.md

```diff
@@ -0,0 +1,311 @@
+# Set up a multi-zone deployment
+
+For a description of how multi-zone deployments work in Kuma, see [about multi-zone deployments](../how-multi-zone-works). This page explains how to configure and deploy Kuma in a multi-zone environment:
+
+- Set up the global control plane
+- Set up the zone control planes
+- Verify control plane connectivity
+- Set up cross-zone communication between data plane proxies
+
+## Set up the global control plane
+
+The global control plane must run on a dedicated cluster, and cannot be assigned to a zone.
+
+:::: tabs :options="{ useUrlFragment: false }"
+::: tab "Kubernetes"
+
+The global control plane on Kubernetes must reside on its own Kubernetes cluster, to keep its CRDs separate from the CRDs the remote control planes create during synchronization.
+
+1.  Run:
+
+    ```bash
+    kumactl install control-plane --mode=global | kubectl apply -f -
+    ```
+
+1.  Find the external IP and port of the `global-remote-sync` service in the `kuma-system` namespace:
+
+    ```bash
+    kubectl get services -n kuma-system
+    NAMESPACE     NAME                   TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)                                                                  AGE
+    kuma-system   global-remote-sync     LoadBalancer   10.105.9.10     35.226.196.103   5685:30685/TCP                                                           89s
+    kuma-system   kuma-control-plane     ClusterIP      10.105.12.133   <none>           5681/TCP,443/TCP,5676/TCP,5677/TCP,5678/TCP,5679/TCP,5682/TCP,5653/UDP   90s
+    ```
+
+    In this example the value is `35.226.196.103:5685`. You pass this as the value of `<global-kds-address>` when you set up the remote control planes.
```

Comment:
The 3685 appears here magically. you might want to reference the network page and mention that by default KDS runs on 3685 no?

Reply:
5685 right? I added reference to ports.